e2e-tests/src/test/java/com/wixpress/common/petri/e2e/FilterParametersConfigTest.java

```diff
@@ -40,16 +40,8 @@ public static void teardown() throws Exception {
 
     @Test
     public void conductingAnExperimentWithGeoFilterWithCustomizedDataExtractor() throws IOException {
-
         addSpec("THE_KEY");
-
         fullPetriClient.insertExperiment(TestBuilders.experimentOnRegisteredWithFilter("THE_KEY").build());
```

Comment:
sorry, i didnt make it clear in the previous comment.
this can be 'experimentOnWinning'.
no real need for logged in user so better not to add cognitive load for someone reading this.